docs/tutorials/index.rst

```diff
@@ -0,0 +1,49 @@
+GeoMesa |release| Tutorials
+===========================
+
```

Comment:
we probably need to have someone go through a verify the tutorials all still work correctly.
***finger on nose***